scripts/eval/yamls/model_gauntlet.yaml

```diff
@@ -112,3 +112,17 @@ model_gauntlet:
     - name: boolq
       num_fewshot: 10
       random_baseline: 0.5
+  - name: programming
+    benchmarks:
+    - name: human_eval
+      num_fewshot: 0
```

Comment:
I know it is not conventional to do few shot prompting, but this seems like a task where models would give more reliable responses with 5-10 in context examples. Could you launch a run with e.g. MPT30B/Falcon 40B with 0, 5, 10 shot of all these benchmarks and see whether perf improves?